src/connector/src/sink/snowflake.rs

```diff
@@ -288,13 +288,14 @@ impl SnowflakeSinkWriter {
         if self.payload.is_empty() {
             return Ok(());
         }
+        let file_suffix = self.file_suffix();
         // todo: change this to streaming upload
```

Comment:
We should have replaced the current implementation with streaming upload in the original PR. Any plan to do it? It shouldn't involve too much work.

Reply:
I'll come with the refactor tomorrow.